deploy.pl

```diff
@@ -36,15 +37,33 @@
 
 my $ua = Mojo::UserAgent->new;
 
+my $application_id = $marathon_json->{id};
+
 my $app_url = URI->new("$marathon_apps_url/".uri_escape($marathon_json->{id}))->canonical->as_string();
+
 my $res = $ua->put($app_url => json => $marathon_json)->res();
 
 if ($res->code != 200 && $res->code != 201) {
     die $res->to_string();
 }
 
+my $deployment_url = URI->new("$marathon_api_url/deployments")->canonical->as_string();
+
+while (!is_deploy_finished($ua, $deployment_url, $application_id)) {
+	print "Waiting for application to be deployed...";
+	sleep(5);
+}
 
 sub is_nonnegative_integer {
     local $_ = shift;
     return /\A\+?\d+\z/
 }
+
+sub is_deploy_finished {
+	my ($ua, $deployment_url, $application_id) = @_;
+	my $res = $ua->get($deployment_url)->res();
+	my $parsed_result = decode_json($res->body);
+	my $applications_being_deployed = $parsed_result->[0]->{affectedApps};
+	
+	return not ("/$application_id" ~~ $applications_being_deployed);
```

Comment:
Since 5.18, perl is throwing warnings about smartmatch (is deprecated). Use for example grep instead it. 
```perl
grep {$_ eq "/$application_id"} @$applications_being_deployed;
```